django/contrib/postgres/forms/jsonb.py

```diff
@@ -0,0 +1,29 @@
+import json
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+__all__ = ['JSONField']
+
+
+class JSONField(forms.CharField):
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be valid JSON."),
+    }
+
+    def __init__(self, **kwargs):
+        kwargs.setdefault('widget', forms.Textarea)
+        super(JSONField, self).__init__(**kwargs)
+
+    def to_python(self, value):
+        try:
+            return json.loads(value)
+        except ValueError:
```

Comment:
If this is loading user-supplied data, this can be made to fairly easily cause other kinds of errors which would cause a different kind of error to propagate upwards. For example:

```python
>>> s = '['*1000 + ']'*1000
>>> json.loads(s)
...
RuntimeError: maximum recursion depth exceeded while decoding a JSON array from a byte string
```

There are probably other kinds of sneaky things we can do here, too. There's an argument to be made that this is OK, that a server crash is the right thing to do here, but I thought I'd point it out.